chromium_src/chrome/browser/ui/views/side_panel/side_panel_entry_key.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_SIDE_PANEL_SIDE_PANEL_ENTRY_KEY_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_SIDE_PANEL_SIDE_PANEL_ENTRY_KEY_H_
+
+#define SidePanelEntryKey SidePanelEntryKey_ChromiumImpl
+
+#include "src/chrome/browser/ui/views/side_panel/side_panel_entry_key.h"  // // IWYU pragma: export
+
+#undef SidePanelEntryKey
+
+#include "brave/components/sidebar/browser/mobile_view_id.h"
+
+// Subclass for handling mobile view panel. As each mobile view panel item
```

Comment:
We should not be implementing this in a way that causes us to have to redefine how this works. Why does each mobile view panel have its own key?

Reply:
W/o unique key, we can't activate mobile view panel via `SidePanelCoordinator`.